main.py

```diff
@@ -2,6 +2,51 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    for genre in genres:
+        Genre.objects.create(name=genre)
+    actors = [("George", "Klooney"),
+              ("Kianu", "Reaves"),
```

Comment:
There is a typo in the actor's name 'Kianu Reaves'. It should be 'Keanu Reeves' to match the correct spelling.